src/csv_arrow_ingest.cpp

```diff
@@ -21,13 +21,13 @@ get_arrow_convert_options(std::vector<std::string> &utf8_columns,
 }
 
 template <typename T>
-std::shared_ptr<arrow::Table> read_csv_stream_to_arrow_table(
-    T &input_stream, std::vector<std::string> &utf8_columns,
-    const std::string &null_value, const std::string &filename) {
+std::shared_ptr<arrow::Table> read_csv_stream_to_arrow_table(T &input_stream, const IngestProperties &props) {
 
   auto read_options = arrow::csv::ReadOptions::Defaults();
+  read_options.block_size = props.csv_block_size_mb << 20;
   auto parse_options = arrow::csv::ParseOptions::Defaults();
-  auto convert_options = get_arrow_convert_options(utf8_columns, null_value);
+  parse_options.newlines_in_values = true;
```

Comment:
sneaking this setting in since it came up in my testing, and I don't see why you'd ever want this to be off.